README.md

```diff
@@ -2,27 +2,32 @@
 
 ## Getting started
 
-### Environment variables
-
-Add your environment variables to `local.settings.json`.
-
-Remember that new environment variables need to be added to the pipeline, so please notify the team about when adding new environment variables.
-
 ### 1. Clone the repository and install dependencies
 
 ```sh
 git clone git@github.com:equinor/data-marketplace-adapter-service.git
 cd data-marketplace-adapter-service
 npm i
+```
+
+### 2. Add environment variables
+
+Add your environment variables to `local.settings.json`. Use `local.settings.template.json` as a guide.
```

Comment:
might be a good idea to highlight that environment variables need to be defined in the `Values` object